.github/workflows/build.yml

```diff
@@ -12,11 +12,14 @@ jobs:
       PLATFORM_USER_NAME: ${{ secrets.PLATFORM_USER_NAME }}
       PLATFORM_USER_PASSWORD: ${{ secrets.PLATFORM_USER_PASSWORD }}
     steps:
+      - name: Setup runner
```

Comment:
Just wondering, could this be run instead inside the various files we have for the e2e-cli, which already does a bunch of "config setup" ?

For example, in `dockerHeadless.sh`, or something similar ?

Reply:
No actually, because you need to do it on the host machine not the docker.
I tried it at first in the dockerfile and it didn't work because the file was read only